content/snowplow-recommendations/supporting-infrastructure/build.md

```diff
@@ -5,60 +5,106 @@ post = ""
 +++
 
 For this step, you have two options:
-#### **Option 1:** Manual
-Follow the linked Snowflake and AWS documentation and create the necessary resources manually.
+#### **1:** Manual
+Follow the linked Snowflake or Databricks and AWS documentation and create the necessary resources manually.
 
 or
 
-#### **Option 2:** Terraform
+#### **2:** Terraform
 Use the provided Terraform script to create the resources. This is the recommended option as it is significantly faster (assuming you're familiar with Terraform).
 {{< tabs >}}
 {{% tab name="Manual" %}}
 #### Option 1: Manual instructions
-**I**. Create an S3 bucket for the data from Snowflake - https://docs.aws.amazon.com/AmazonS3/latest/user-guide/create-bucket.html
+
+##### AWS
+
+**I**. Create an S3 bucket for the data from the dbt package - https://docs.aws.amazon.com/AmazonS3/latest/user-guide/create-bucket.html
 
 **II**. Follow the instructions in the following link to configure a storage integration between Snowflake and S3 - https://docs.snowflake.com/en/user-guide/data-load-s3-config-storage-integration.html. You will need either global CREATE STORAGE INTEGRATION permissions or to be an ACCOUNTADMIN in Snowflake for this step, as well as permissions in AWS to create and manage IAM policies and roles. 
-**Note: The data from Snowflake will be loaded to the root of your bucket, so don't add any prefixes to the resource name when creating the policies.**
 
 **III**. Create a policy and role for AWS personalize - follow instructions here https://docs.aws.amazon.com/personalize/latest/dg/set-up-required-permissions.html 
 
```

Comment:
Why did you delete the note about the confused deputy problem? I think it's not super obvious from the set-up-required-permissions page that it should be done